apps/web/src/components/settings/sections/editor-icons.tsx

```
@@ -0,0 +1,32 @@
+import type { ComponentProps } from 'react';
+
+type IconProps = ComponentProps<'svg'>;
+
+/**
+ * Cursor editor logo icon
+ */
+export function CursorIcon(props: IconProps) {
+  return (
+    <svg viewBox="0 0 24 24" fill="currentColor" xmlns="http://www.w3.org/2000/svg" {...props}>
```

Biome (2.3.14)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)